content/account/dashboard/console.textile

```diff
@@ -0,0 +1,80 @@
+
+h2. Channels
+
+This following provides step-by-step instructions for connecting to a channel, publishing messages, managing presence, viewing channel and presence history, interacting with the Ably channel via a command-line tool, and controlling channel operations.
+
+h3. Connect to a channel
+
+The following explains how to connect to a channel:
+
+|_. Step |_. Action |
+| *Enter a channel name* | In the channel name field, choose a name (e.g get-started). |
+| *Attach to channel* | Click the *attach to channel* button. This connects you to the *get-started* channel, enabling you to start publishing or subscribing to messages. |
+| *Monitor channel status* | The interface will display the channel status as *pending* and then *attached* once connected, confirming that the channel is ready for interaction.|
+
+h3. Publish a message
+
+The following explains how to publish a message:
+
+|_. Step |_. Action |
+| *Message data* | In the *message data* field, type a message (e.g. example). |
+| *Publish message* | Click the *publish message* button to send the message to the *get-started* channel. |
+| *View the message* | If you have a subscriber , it will receive and display the message in the console. |
+
+The following explains how to interact with presence:
+
+|_. Step |_. Action |
+| *Client ID* | Enter a unique client ID to simulate joining the presence of the channel. |
+| *Enter presence* | Click *enter presence* to indicate that this client is now in the channel. |
+| *Monitor presence* | The interface will list all clients in the channel under *presence members*. |
+
+h3. View channel and presence history
+
+The following explains how to view channel and presence history:
+
+|_. Step |_. Action |
+| *Channel history* | Click *channel history* to see a log of all past messages sent in the channel. |
+| *Presence history* | Click *presence history* to view a log of clients entering and leaving the channel. |
+
+h3. Interact with the Ably channel using command-line
+
+The following explains how to interact with the Ably channel using a command-line tool like cURL:
+
+|_. Step |_. Action |
+| *Open terminal* | Open a terminal or command prompt on your computer.|
+| *Copy the cURL command* | Copy the following command: |
+| *Enter event name* | In the *event name* field, type *publish*, or any message you want to send.|
+| *Enter message data* | Type the message data you want to send.|
+| *Execute the command* | Paste the command into your terminal and press enter. This will publish a message with the name *publish* and data *example* to the *get-started* channel.|
+| *Check the output* | If successful, any subscriber to the *get-started* channel should receive and display the message.|
+
+h3. Control the channel
+
+The following explains how to control the channel:
+
+|_. Step |_. Action |
+| *Detach* | Click *detach* to disconnect from the channel. |
+| *Pause* | Use *pause* to temporarily stop receiving messages. |
```

Comment:
Might be worth clarifying this is only in the dev console, not the whole app 🙂

Reply:
Not 100% on this fix.